package org.autorepair.data.exceptions

class Error: Throwable(message = "ERROR")